ethcore/light/src/net/mod.rs

```diff
@@ -157,6 +158,54 @@ pub struct Peer {
 	awaiting_acknowledge: Option<(SteadyTime, Arc<FlowParams>)>,
 }
 
+/// Whether or not a peer was kept by a handler
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum PeerStatus {
```

Comment:
agree, true for kept (positive result of operation) and false for dropped (negative result) is pretty natural to assume though